opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -701,6 +701,38 @@ def zip_model(
             )
         print("zip file is saved to " + zip_file_path + "\n")
 
+    def fix_truncation(
+        self,
+        save_json_folder_path: str,
+        max_length: int,
+    ) -> None:
+        """
+        Description:
```

Comment:
And say “Fill truncation field in tokenizer.json when it is null” here instead so other people know what exactly this function addresses without reading the code